modules/test/playwright/tests/account-admin-web/accountPermissions.spec.ts

```diff
@@ -311,3 +315,118 @@ test.describe('Test for Organization Account visibility depending on Permissions
 		await expect(accountsPage.newButton).toHaveCount(0);
 	});
 });
+
+test.describe('This test asserts that an Account Admin can add and unassign an organization to an account', () => {
+	test('LPD-30009 Account admin can unassign organization to account', async ({
+		accountManagementWidgetPage,
+		accountOrganizationsPage,
+		accountsPage,
+		apiHelpers,
+		page,
+	}) => {
+		page.on('dialog', (dialog) => dialog.accept());
+
+		const userName = 'demo.unprivileged';
+		const accountAdministrator = 'Account Administrator';
+
+		await page.goto('/search');
+
+		await accountManagementWidgetPage.addAccountManagementWidget();
+
+		try {
+			const account = await apiHelpers.headlessAdminUser.postAccount({
+				name: 'Account' + getRandomInt(),
+				type: 'business',
+			});
+
+			apiHelpers.data.push({id: account.id, type: 'account'});
+
+			await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+				account.id,
+				[`${userName}@liferay.com`]
+			);
+
+			const rolesResponse =
+				await apiHelpers.headlessAdminUser.getAccountRoles(account.id);
+
+			const accountAdminRole = rolesResponse?.items?.filter(
+				(role) => role.name === accountAdministrator
+			);
+
+			const userAccount =
+				await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+					`${userName}@liferay.com`
+				);
+
+			await apiHelpers.headlessAdminUser.assignUserToAccountRole(
+				account.id,
+				accountAdminRole[0].id,
+				userAccount.id
+			);
+
+			const organization =
+				await apiHelpers.headlessAdminUser.postOrganization();
+
+			await apiHelpers.headlessAdminUser.assignAccountToOrganization(
+				account.id,
+				organization.id
+			);
+
+			await performLogout(page);
+			await performLogin(page, `${userName}`);
+
+			await page.goto('/search');
+
+			await page.getByText(account.name).click();
+			await accountsPage.organizationsTab.click();
+
+			await accountManagementWidgetPage.waitForSearchInput();
+
+			await expect(accountsPage.newButton).toHaveCount(0);
+
+			await performLogout(page);
+			await performLogin(page, 'test');
+
```

Comment:
please remove this: we are complicating the test without any reason: there should be already a test for this